Src/Sarif.PatternMatcher.Function/SpamAnalyzer.cs

```diff
@@ -15,6 +16,7 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Function
     internal static class SpamAnalyzer
     {
         internal static readonly IFileSystem FileSystem;
+        internal static ISet<Skimmer<AnalyzeContext>> Skimmers;
```

Comment:
>internal static ISet<Skimmer<AnalyzeContext>> Skimmers [](start = 8, length = 54)

Added a naive cache to prevent loading rules all the time. Since the rules won't change unless we publish a new version, that can be read once.